programs/local/LocalServer.cpp

```diff
@@ -756,19 +756,7 @@ void LocalServer::addOptions(OptionsDescription & options_description)
 
         /// If structure argument is omitted then initial query is not generated
         ("structure,S", po::value<std::string>(), "structure of the initial table (list of column and type names)")
-        ("file,f", po::value<std::string>(), "path to file with data of the initial table (stdin if not specified)")
-
-        ("input-format", po::value<std::string>(), "input format of the initial table data")
-        ("output-format", po::value<std::string>(), "default output format")
-
-        ("logger.console", po::value<bool>()->implicit_value(true), "Log to console")
-        ("logger.log", po::value<std::string>(), "Log file name")
-        ("logger.level", po::value<std::string>(), "Log level")
-
-        ("no-system-tables", "do not attach system tables (better startup time)")
-        ("path", po::value<std::string>(), "Storage path")
-        ("only-system-tables", "attach only system tables from specified path")
-        ("top_level_domains_path", po::value<std::string>(), "Path to lists with custom TLDs")
+        ("*if you want to have more info, you can go to https://clickhouse.com/docs/*", po::value<std::string>(), "")
```

Comment:
Maybe "All setting parameters are documented in detail at https://clickhouse.com/docs/"?